test/tools/jsontests/LegacyTestsBoost.cpp

```diff
@@ -0,0 +1,220 @@
+#include <test/tools/jsontests/BlockChainTests.h>
+#include <test/tools/jsontests/StateTests.h>
+#include <test/tools/libtesteth/TestHelper.h>
+
+#include <boost/test/unit_test.hpp>
+
+using namespace dev::test;
+namespace fs = boost::filesystem;
+
+fs::path StateTestSuiteLegacyConstantinople::suiteFolder() const
+{
+    if (Options::get().fillchain || Options::get().filltests)
+        BOOST_FAIL("Legacy tests are sealed and not refillable!");
```

Comment:
That's not a great idea to throw from a const getter method. If you really need this, you should check this on the calling side.

Reply:
which would be a test fixture. no template then